java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -181,4 +191,28 @@ public CompletableFuture<String> set(
         String[] arguments = ArrayUtils.addAll(new String[] {key, value}, options.toArgs());
         return commandManager.submitNewCommand(SetString, arguments, this::handleStringOrNullResponse);
     }
+
+    @Override
+    public CompletableFuture<String> hget(@NonNull String key, @NonNull String field) {
+        return commandManager.submitNewCommand(
+                HashGet, new String[] {key, field}, this::handleStringOrNullResponse);
+    }
+
+    @Override
+    public CompletableFuture<Long> hset(
+            @NonNull String key, @NonNull Map<String, String> fieldValueMap) {
+        String[] args =
+                Stream.concat(
+                                Stream.of(key),
+                                fieldValueMap.entrySet().stream()
+                                        .flatMap(entry -> Stream.of(entry.getKey(), entry.getValue())))
+                        .toArray(String[]::new);
```

Comment:
Move this to utils

Reply:
Will add this in PR https://github.com/aws/glide-for-redis/pull/955